content/sensu-core/0.29/reference/filters.md

```diff
@@ -295,6 +296,135 @@ value), and the event data does not contain a matching [eval token
 attribute][13], an log entry indicating an error called `"filter eval unmatched
 tokens"` will be published to the Sensu server log._
 
+## Built-in Filters
+### Occurrences filter {#built-in-filters-occurrences}
+
+The occurrences filter is included in every install of Sensu.
+It lets you control the number of duplicate events that reach the handler.
+You can apply the occurrences filter to a handler using the `filters` handler definition attribute.
+
+Here's an example of a handler definition that uses the occurrences filter:
+
+{{< highlight json >}}
+{
+  "handlers": {
+    "email": {
+      "type": "pipe",
+      "command": "email.rb",
+      "filters": ["occurrences"]
+    }
+  }
+}
+{{< /highlight >}}
+
+#### Occurrences filter attributes {#occurrences-filter-attributes}
+
+The occurrences filter lets you configure the number of occurrences and the refresh interval at the check level using two check definition attributes: `occurrences` and `refresh`.
+
+Here's an example of a check definition that passes events to the `email` handler starting with the third occurrence every 60 minutes:
```

Comment:
Wouldn't this be `starting with the second occurrence every 60 minutes`? The JSON example has occurrences set to 2, so I believe it would be handled after 2 occurrences. I can test this if we aren't sure.

Reply:
Thanks! That was exactly what I was wondering